deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -974,6 +994,213 @@ def generate_event(self):
         return event_final
 
 
+class GeneratorVulnerabilityEvents:
```

Comment:
We need to also change the GeneratorSyscollector class, avoiding to duplicate code, removing vulnerable package logic.